src/data.js

```diff
@@ -10,8 +10,10 @@ export function create (cozy, doctype, attributes) {
     if (isV2) {
       attributes.docType = doctype
     }
-    const path = createPath(cozy, isV2, doctype)
-    return cozyFetchJSON(cozy, 'POST', path, attributes).then((resp) => {
+    const path = createPath(cozy, isV2, doctype, attributes._id)
```

Comment:
Are you sure it works when there is no `_id`? It looks like `createPath` is expecting a blank string, not null. Maybe:

```js
const path = createPath(cozy, isV2, doctype, attributes._id || '')
```